EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76595
+thelocal.ch#?#div[class*="section-sponsored"]:contains(Sponsored)
```

Comment:
Is extended CSS required?

Btw I am not sure if it must be hidden, It looks just as articles. Can be interesting for some people.

Blocked social share.

Reply:
I will move it to annoyances with simple ule.